src/tests/ftest/util/server_utils.py

```diff
@@ -48,6 +53,193 @@ class ServerFailed(Exception):
     """Server didn't start/stop properly."""
 
 
+class ServerCommand(CommandWithParameters):
+    """Defines a object representing a server command."""
+
+    def __init__(self, hosts):
```

Comment:
Given this error:
```
06:29:19 INFO | Running '/bin/sudo -n -s daos_server -o /var/lib/jenkins/jenkins-1/docker_1/workspace/daos-stack_daos_PR-911@2/src/tests/ftest/data/daos_avocado_test.yaml -t 1'
06:29:19 DEBUG| [stderr] /bin/bash: daos_server: command not found
```
It would appear that a path to `daos_server` is required.  One option would be to add an optional `path` argument to the init, which, if specified, could be used to update the `self._command` attribute with `os.path.join()`:

Reply:
Ok. Will add that in there. I'll add the path argument on the `CommandWithParameters` class to extend that functionality to other commands and not only `DmgCommand` and `ServerCommand`.